plugins/templates/create.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const boom = require('boom');
+const schema = require('screwdriver-data-schema');
+const hoek = require('hoek');
+const urlLib = require('url');
+
+module.exports = () => ({
+    method: 'POST',
+    path: '/templates',
```

Comment:
If you know exactly what you're looking for, why be opposed to being a RESTful API and not do `/templates/<templateName>/version/<version>`?

What makes that different than `/templates/<templateName>` for listing all the versions and `/templates/<templateName>?version=2` for listing all versions >= 2.0? Or even `/templates/<templateName>?labels=stable`? 

This is nothing special to what other services we own. Many services cover this already; what makes *this* API so special that it's breaking from generally accepted principles?